download_invoices.py

```diff
@@ -109,7 +138,12 @@ def get_all_orders_with_tracking_info(amount_of_invoices):
         print(f'Going to click track package link number {i+1}')
         progress_tracker = browser.find_elements_by_xpath("//a[contains(@href, 'progress-tracker')]")[i]
         progress_tracker.click()
-        time.sleep(config.WAITING_TIME_BETWEEN_PAGES)
+
+        try:
+            wait = WebDriverWait(browser, config.WAITING_TIME_BETWEEN_PAGES)
```

Comment:
i see the same pattern with `WebDriverWait` in many places... you can make a helper method.

Reply:
just to simplify your life